.github/workflows/tests.yml

```diff
@@ -16,18 +16,20 @@ jobs:
       - name: Download WSI artifacts
         id: load-wsis
         run: |
-          mkdir -p tests/fixtures/external_wsi
-          wget http://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/CMU-3.svs -P tests/fixtures/external_wsi
+          mkdir -p tests/fixtures/external-svs
+          wget http://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/CMU-3.svs -P tests/fixtures/external-svs
+          mv tests/fixtures/external-svs/CMU-3.svs tests/fixtures/external-svs/cmu-3.svs
```

Comment:
i think this line can be avoided using the ` -O example.html` option for wget. So after the `-O` option you should pass the actual name you want.